### PR TITLE
Article Updated Date

### PR DESCRIPTION
**Pull request by jestapinski**
This PR adds support for the "Updated Date" field for an article. I need to double check with Sophie the exact name of this field.

Screen Shot 2020-02-27 at 6 04 26 PM

On extremely small screens, let's stack the text to avoid odd wraps.

Screen Shot 2020-02-27 at 6 04 36 PM

### CODE REVIEW

File: src/components/dev-hub/blog-post-title-area.js [hunk 1]
```diff
@@ -18,26 +19,54 @@ const PostMetaLine = styled('div')`
...
+    }
+`;
+
 const DateText = styled(P)`
-    margin-right: ${size.medium};
+    margin-right: ${size.tiny};
     @media ${screenSize.upToLarge} {
         font-size: ${fontSize.xsmall};
     }
+    ${({ withBullet }) => withBullet && bulletStyling};
```
Comment (greggb) [inline src/components/dev-hub/blog-post-title-area.js]: Bet we could do this with last-child or the `+` selector, but mostly just a noit

Comment (jestapinski) [reply src/components/dev-hub/blog-post-title-area.js]: Only issue there is if the updated date is not provided we don't want the bullet on the initial published date. I would like to keep as-is?

**Comment by jestapinski** [comment]
Thanks!

File: src/components/dev-hub/blog-post-title-area.js (hunk 1 above)
Comment (greggb) [reply src/components/dev-hub/blog-post-title-area.js]: Sure you can keep, but that can be done with css :)

**Pull request merged**